src/Lean/Util/FindExpr.lean

```diff
@@ -8,6 +8,21 @@ import Lean.Expr
 namespace Lean
 namespace Expr
 
+/-- Run the given `visit` function on each _immediate_ child subexpression of the given expression.
+It will fail if there are no subexpressions.
+
+Does not instantiate bound variables if the subexpression is below a binder.
+Usage with monads deriving from `MetaM` is not recommended.
+-/
+def findChildren [Alternative m] (visit : Expr → m α) : Expr → m α
```

Comment:
possibly silly question -- where can I see the generated code?

Reply:
If you want to see the IR for a function without ref-count instructions. You can add
```
set_option trace.compiler.ir.init true
```
The output is a bit hard to read since it contains new functions that have been lambda lifted, ground constants that have been extracted, and code that has been specialized.